exercises/accumulate/accumulate.py

```diff
@@ -1,2 +1,2 @@
-def accumulate():
+def accumulate(array, function):
```

Comment:
This looks Ok to me, but some people would argue that you will force people to certain implementations by giving exact params.

I personally think it's a good idea to do so anyway, if one still doesn't want to follow suggested method signature he/she can ignore it and write his/her own implementation.

Moreover, sometimes forcing certain implementation helps people to use features of the language they aren't used to (like passing functions to a method, not all languages allow that etc)

As an improvement, I would name params with meaningful names rather than generic ones. But it's up for you to decide